src/ReactLeafletMap/index.js

```diff
@@ -123,7 +123,9 @@ class ReactLeafletMap extends React.PureComponent {
             }
 
             // TODO for IndexedVectorLayer rerender (see IndexedVectorLayer render method)
-            this.setState({viewport});
+            if (!_.isEqual(viewport, this.state.viewport)) {
```

Comment:
Is _.isEquaal good for performance?

Reply:
It's a simple object, so the impact on the preformance should be minimal